Content.Client/GameObjects/Components/Weapons/Melee/MeleeWeaponArcAnimationComponent.cs

```diff
@@ -44,6 +47,11 @@ internal void Update(float frameTime)
                 Vector4.Clamp(_meleeWeaponAnimation.Color + _meleeWeaponAnimation.ColorDelta * _timer, Vector4.Zero, Vector4.One);
             _sprite.Color = new Color(r, g, b, a);
 
+            if (_attacker != null && _attacker.IsValid())
+            {
+                Owner.Transform.GridPosition = _attacker.Transform.GridPosition;
```

Comment:
It's probably better to parent the animation entity to the attacker itself so that you don't have to move it around in Update and the transform system just handles it, instead.

Reply:
Done